src/20250421_Multi_GHOGnosisLaunch/AaveV3Base_GHOGnosisLaunch_20250421.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {IUpgradeableBurnMintTokenPool_1_5_1} from 'src/interfaces/ccip/tokenPool/IUpgradeableBurnMintTokenPool.sol';
+import {IRateLimiter} from 'src/interfaces/ccip/IRateLimiter.sol';
+import {GhoBase} from 'aave-address-book/GhoBase.sol';
+import {CCIPUtils} from './utils/CCIPUtils.sol';
+import {GHOLaunchConstants} from './utils/GHOLaunchConstants.sol';
+
+/**
+ * @title GHO Gnosis Launch
+ * @author kpk
+ * @notice This proposal is used to launch GHO on Gnosis Chain
+ * - Discussion: https://governance.aave.com/t/arfc-launch-gho-on-gnosis-chain/21379
+ * - Snapshot: https://snapshot.box/#/s:aavedao.eth/proposal/0x62996204d8466d603fe8c953176599db02a23f440a682ff15ba2d0ca63dda386
+ */
+contract AaveV3Base_GHOGnosisLaunch_20250421 is IProposalGenericExecutor {
+  uint64 public constant GNOSIS_CHAIN_SELECTOR = CCIPUtils.GNOSIS_CHAIN_SELECTOR;
+
+  // https://basescan.org/address/0x6Bb7a212910682DCFdbd5BCBb3e28FB4E8da10Ee
```

Comment:
This is incorrect. I suggest to remove comments these kind of links across files. It's error-prone and we are using the Constants file anyway so this is pointless.